CMake/config.h.cmake.in

```diff
@@ -1,204 +1,216 @@
-#define HAVE_MEMMOVE 1
-#endif
-
-/* Define to 1 if you have the <memory.h> header file. */
-#define HAVE_MEMORY_H 1
-
-/* Support for NetCDF */
-#define HAVE_NETCDF_DRIVER
-
-/* Support for PDB Proper */
-/* #undef HAVE_PDBP_DRIVER */
-
-/* Support for PDB */
-#define HAVE_PDB_DRIVER 1
-
-/* Define to 1 if you have the <readline.h> header file. */
-/* #undef HAVE_READLINE_H */
-
-/* Define if your readline library has \`add_history' */
-/* #undef HAVE_READLINE_HISTORY */
-
-/* Define to 1 if you have the <readline/history.h> header file. */
-/* #undef HAVE_READLINE_HISTORY_H */
-
-/* Define to 1 if you have the <readline/readline.h> header file. */
-/* #undef HAVE_READLINE_READLINE_H */
-
-/* System provides setvbuf prototypes */
-#define HAVE_SETVBUF_POINTER 1
-
-/* Define to 1 if you have the <stdarg.h> header file. */
-#define HAVE_STDARG_H 1
-
-/* Define to 1 if you have the <stdint.h> header file. */
-/* #undef HAVE_STDINT_H */
-
-/* Define to 1 if you have the <stdlib.h> header file. */
-#define HAVE_STDLIB_H 1
-
-/* Define to 1 if you have the `strerror' function. */
-#define HAVE_STRERROR 1
-
-/* Define to 1 if you have the <strings.h> header file. */
-/* #undef HAVE_STRINGS_H */
-
-/* Define to 1 if you have the <string.h> header file. */
-#define HAVE_STRING_H 1
-
-/* Define to 1 if you have the <sys/fcntl.h> header file. */
-/* #undef HAVE_SYS_FCNTL_H */
-
-/* Define to 1 if you have the <sys/stat.h> header file. */
-#define HAVE_SYS_STAT_H 1
-
-/* Define to 1 if you have the <sys/time.h> header file. */
-/* #undef HAVE_SYS_TIME_H */
-
-/* Define to 1 if you have the <sys/types.h> header file. */
-#define HAVE_SYS_TYPES_H 1
-
-/* Support for Taurus */
-#define HAVE_TAURUS_DRIVER 1
-
-/* Define to 1 if you have the <unistd.h> header file. */
-/* #undef HAVE_UNISTD_H */
-
-/* Define to 1 if you have the <zlib.h> header file. */
-#define HAVE_ZLIB_H 1
-
-/* Name of package */
-#define PACKAGE "silo"
-
-/* Define to the address where bug reports for this package should be sent. */
-#define PACKAGE_BUGREPORT "visit-help@llnl.gov"
-
-/* Define to the full name of this package. */
-#define PACKAGE_NAME "silo"
-
-/* Define to the full name and version of this package. */
-/* #undef PACKAGE_STRING */
-
-/* Define to the one symbol short name of this package. */
-#define PACKAGE_TARNAME "silo"
-
-/* include the file that has the version defines */
-#include "siloversion.h"
-
-/* `off64_t' does not exist on windows, so cannot take 'sizeof' */
-/*  Set it to '4' if building for a 32-bit system, and '8' if */
-/*  building for 64-bit. */
-#ifdef _WIN64
-  #define SIZEOF_OFF64_T 8
-#else
-  #define SIZEOF_OFF64_T 4
-#endif
-
-/* Define to 1 if you have the ANSI C header files. */
-#define STDC_HEADERS 1
-
-/* Define to 1 if the X Window System is missing or not being used. */
-/* #undef X_DISPLAY_MISSING */
-
-/* Override longjmp */
-/* #undef longjmp */
-
-/* Override setjmp */
-/* #undef setjmp */
+/* Took config/config.h.in and cmakeified */
```

Comment:
This looks like its based from original `SiloWindows/include/config.h` and I just wanna confirm you didn't find anything missing here that might have been in `config/config.h.in`?

Reply:
config.h.cmake.in was made from config/config.h.in.